hello-world.c

```diff
@@ -0,0 +1,16 @@
+#include <stdio.h> 
+void main() 
+{
+  int a,b;
+  printf("enter the numbers to swap");
+  scanf("%d\n%d",&a,&b);
+  printf("The numbers before swapping are a=%d and b=%d",&a,&b);
+  swap(&a,&b); printf("After swapping the numbers are a=%d and b=%d",&a,&b);  
+} 
+void swap(int *a,int *b) 
+{   
+  int temp;
+  temp=*b; 
+  *b=*a; 
+  *a=temp; 
+}
```

Comment:
This code will not work. Read all the Linux code and change all the variable names. Ask Dheeraj for help, show him your repo. Then make necessary changes accordingly and raise a pull request again.